src/main/com/fulcrologic/rad/type_support/date_time.cljc

```diff
@@ -259,13 +267,16 @@
    (inst->html-datetime-string *current-zone-name* inst))
   ([zone-name inst]
    [(? ::zone-name) (? inst?) => string?]
+   (inst->html-datetime-string zone-name inst))
+  ([zone-name inst default-value]
+   [(? ::zone-name) (? inst?) (? inst?) => string?]
    (try
      (let [z         (get-zone-id (or zone-name "UTC"))
-           ldt       (ldt/of-instant (inst->instant (or inst (now))) z)
+           ldt       (ldt/of-instant (inst->instant (or inst default-value)) z)
            formatter dtf/iso-local-date-time]
        (ldt/format ldt formatter))
      (catch #?(:cljs :default :clj Exception) e
-       nil))))
+       ""))))
```

Comment:
should this be `default-value`?  Torn on this one. It failed, so empty probably makes more sense, but I'm not sure what a default-value means in this case. I guess the docstring can define it, but once we publish, it'll sorta be set  in stone.

Reply:
Hm it seems like we have two different ideas what the default value is. I used it to patch around the input value `inst` defaulting to `(now)`, so `default-value` is used as an input, e.g. in function `inst->local-date` the `default-value` needs to be an `inst`, not `local-date`. So that wouldn't apply here.

But you are right, it would be more appropriate that the `default-value` were in the type of the return. But that complicates matters for the purpose this patch was intended, which is that the date controls could actually have `nil` as a possible initial state.

Perhaps I should just handle that in the component itself and avoid all these complications in RAD library.

Reply:
Yeah, it's a little non-standard, and therefore potentially confusing. I the inst could not be formatted it is an indication of a bug, and should never really happen. So, I think giving back a blank string is fine. Let's go with it as-is.